react/components/MiniCartContent.js

```diff
@@ -151,23 +151,35 @@ class MiniCartContent extends Component {
     })
   }
 
+  sumTotalPriceWithAttachs = (addedOptions) => {
+    return addedOptions.reduce((acc, option) =>  acc + option.sellingPrice * option.quantity, 0)
+  }
+
+  createOptionShapeFromItem = (option) => ({
+    ...this.createProductShapeFromItem(option),
+    isSingleChoice: isSingleChoiceOption(option, this.props.data.orderForm),
+    optionType: option.parentAssemblyBinding.split('_')[1],
```

Comment:
nesse momento tu tem certeza que `option` e `parentAssemblyBinding` nao vao ser undefined e vai conter um `_`?

Reply:
I like the use of Ramda in the util file, perhaps you could include it here. Ramda's `path` and dot-free programming would avoid possible errors